ds_digital_ads/pipeline/recent_search_twitter.py

```diff
@@ -0,0 +1,293 @@
+"""
+Script to collect Twitter data from the past 7 days using the recent search endpoint, given a set of rules and query parameters.
+If data on a specific ruleset has been collected sometime in the past 7 days, only new data gets collected.
```

Comment:
I always find it helpful to add a line about how to run the script at the top if i ever need to rerun

```suggestion
Script to collect Twitter data from the past 7 days using the recent search endpoint, given a set of rules and query parameters.
If data on a specific ruleset has been collected sometime in the past 7 days, only new data gets collected.

python ds_digital_ads/pipeline/recent_search_twitter.py
```

Reply:
Love this, I will do it in other places too! Thanks for the suggestion :)